theano/gradient.py

```diff
@@ -1201,58 +1201,62 @@ def try_to_copy_if_needed(var):
                         is_zero = _is_zero(term)
                         assert is_zero in ['yes', 'no', 'maybe']
                         if is_zero == 'maybe':
-                            msg = "%s.grad returned %s of type %s for input"
-                            msg += " %d. This input's only connections to "
-                            msg += "the cost through this op are via "
-                            msg += "integer-valued outputs so it should be "
-                            msg += "NullType, DisconnectedType, or some form "
-                            msg += "of zeros. It is not NullType or "
-                            msg += "DisconnectedType and theano can't "
-                            msg += "simplify it to a constant, so it's not "
-                            msg += "verifiably zeros."
-
-                            msg = msg % (str(node.op), str(term),
-                                         str(type(term)), i)
-
-                        if is_zero == 'no':
-                            msg = "%s.grad returned %s of type %s for input"
-                            msg += " %d. Since this input is only connected "
-                            msg += "to integer-valued outputs, it should "
-                            msg += "evaluate to zeros, but it evaluates to"
-                            msg += "%s."
-
-                            msg % (node.op, term, type(term), i,
+                            msg = "%s.grad returned %s of type %s for input" \
+                                " %d. This input's only connections to " \
+                                "the cost through this op are via " \
+                                "integer-valued outputs so it should be " \
+                                "NullType, DisconnectedType, or some form " \
+                                "of zeros. It is not NullType or " \
+                                "DisconnectedType and theano can't " \
+                                "simplify it to a constant, so it's not " \
+                                "verifiably zeros."
+
+                            msg %= (node.op, term, type(term), i)
+
+                        elif is_zero == 'no':
+                            msg = "%s.grad returned %s of type %s for input" \
+                                " %d. Since this input is only connected " \
+                                "to integer-valued outputs, it should " \
+                                "evaluate to zeros, but it evaluates to" \
+                                "%s."
```

Comment:
Normally, we don't use `\` anymore in the code. If you can revert this, it would be great.